usecase/src/main/java/org/mobilitydata/gtfsvalidator/usecase/GenerateExclusionFilenameList.java

```diff
@@ -42,25 +39,26 @@ public GenerateExclusionFilenameList(final GtfsSpecRepository gtfsSpecRepo,
      * Use case execution method: returns the list of filename on which the GTFS validation should not be
      * applied to.
      */
-    public List<String> execute() {
-        final String[] toExcludeFromValidation =
-                execParamRepo.getExecParamValue(ExecParamRepository.EXCLUSION_KEY)
+    public ArrayList<String> execute() {
+        final List<String> toExcludeFromValidation =
+                Arrays.asList(execParamRepo.getExecParamValue(ExecParamRepository.EXCLUSION_KEY)
                         .replace("[", "")
                         .replace("]", "")
-                        .split(",");
+                        .split(","));
 
-//        final List<String> gtfsFilenameList = new ArrayList<>();
-//        gtfsFilenameList.addAll(gtfsSpecRepo.getRequiredFilenameList());
-//        gtfsFilenameList.addAll(gtfsSpecRepo.getOptionalFilenameList());
+        final List<String> gtfsFilenameList = new ArrayList<>();
+        gtfsFilenameList.addAll(gtfsSpecRepo.getRequiredFilenameList());
+        gtfsFilenameList.addAll(gtfsSpecRepo.getOptionalFilenameList());
 
-        // TODO: check validity of user input
-
-        final GtfsNode root = gtfsSpecRepo.getGtfsDependencySchema();
+        if (!gtfsFilenameList.containsAll(toExcludeFromValidation)) {
+            throw new IllegalArgumentException("Filename list to exclude is malformed: " + toExcludeFromValidation);
```

Comment:
Clarify error message to be explicit about the fact a file was marked as to be excluded while the spec doesn't define a file with such a name